README.md

```diff
@@ -143,22 +143,21 @@ Since the resource state is stored outside of the container, the GPU cluster lif
 ## Instructions
 
 1. gcloud auth application-default login.
-2. gcloud auth configure-docker us-docker.pkg.dev
```

Comment:
The purpose of this was, until you configure docker for a project gcp does not allow you to pull images. But if that is not the case then we can remove this line. 

The `configure-docker` is needed only once per project I think.

Reply:
I didn't know that we don't need `configure-docker` for public images. We can definitely remove this